src/components/form/form.scss

```diff
@@ -5,6 +5,8 @@
 
 /**
  * @prop --form-body-padding: space around content of the form
+ * @prop --form-background-color-of-even-settings-rows: Background of even rows in the form, when layout type is `row`. Defaults to `transparent`.
+ * @prop --form-background-color-of-odd-settings-rows:Background of odd rows in the form, when layout type is `row`. Defaults to `--contrast-200`.
```

Comment:
We should remove the `settings` substring from these

Reply:
Not sure why this was resolved without a fixup :smile:

Reply:
😮 WTF. I'm 500% sure I fixed and pushed it 😓

Reply:
good catch!

Reply:
⚡